RiotSharpTest/TournamentRiotApiTest.cs

```diff
@@ -74,26 +71,19 @@ public void GetTournamentMatch_Test()
         [TestCategory("TournamentRiotApi"), TestCategory("Async")]
         public void CreateProviderAsync_CreateTournamentAsync_CreateTournamentCodeAsync_UpdateTournamentCodeAsync_Test()
```

Comment:
Rebase is done and now I know why the test fails. It fails because of an exceeded rate limit,.